mixins/form.jsx

```diff
@@ -411,7 +417,13 @@ module.exports = {
         submitting: true
       });
       submitProps = form.buildProps(props);
-      this.submit("/api/signup", submitProps, this.signupSuccess, this.signupError);
+      this.submit(url, submitProps, success, error);
     }
+  },
+  signup: function(validate, props) {
+    this.doSignup("/api/signup", validate, props, this.signupSuccess, this.signupError);
```

Comment:
I feel like this can be confusing and that's why I suggested at least naming it with a specific name in the path to make it clear that this signup path is for `basket` and the other `mailchimp`. Right now, if anyone who's not familiar with the repo and looking at the `/api/signup` would not be able to tell. Also, `/api/mailchimp` is an ugly route name in my opinion... I feel like the route name is not descriptive enough.